### PR TITLE
README update due to recent improvements

### PR DESCRIPTION
**Pull request by akolotov**
Recently two new parameters were introduced:
- `APP_STYLES` allows to specify a theme prepared for particular deployment (https://github.com/poanetwork/bridge-ui/pull/192)
- `REACT_APP_TITLE` allows to specify the page title (https://github.com/poanetwork/bridge-ui/pull/201)

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -29,10 +28,11 @@ REACT_APP_FOREIGN_GAS_PRICE_FALLBACK=5000000000
 REACT_APP_FOREIGN_GAS_PRICE_UPDATE_INTERVAL=15000
 
 # Default
-REACT_APP_DESCRIPTION="The POA cross-chain bridge serves as a method of transferring POA native tokens from the POA Network to the Ethereum network in a quick and cost-efficient manner."
+REACT_APP_TITLE=TokenBridge UI app - %c
+REACT_APP_DESCRIPTION=The POA cross-chain bridge serves as a method of transferring POA native tokens from the POA Network to the Ethereum network in a quick and cost-efficient manner.
 
 # RSK
-#REACT_APP_DESCRIPTION="The TokenBridge serves as a method of transferring Bancor Network tokens between the Ethereum network to Rootstock network in a quick and cost-efficient manner."
+#REACT_APP_DESCRIPTION=The TokenBridge serves as a method of transferring Bancor Network tokens between the Ethereum network to Rootstock network in a quick and cost-efficient manner.
```
Comment (andogro) [inline .env.example]: The TokenBridge serves as a method of transferring Bancor Network tokens between the Ethereum network and the Rootstock network in a quick and cost-efficient manner.

File: README.md [hunk 2]
```diff
@@ -179,7 +179,9 @@ REACT_APP_FOREIGN_GAS_PRICE_ORACLE_URL | The URL used to get a JSON response fro
 REACT_APP_FOREIGN_GAS_PRICE_SPEED_TYPE | Gas Price speed (slow, standard, fast, instant)
 REACT_APP_FOREIGN_GAS_PRICE_FALLBACK | The gas price (in Wei) that is used if both the oracle and the fall back gas price specified in the Foreign Bridge contract are not available.
 REACT_APP_FOREIGN_GAS_PRICE_UPDATE_INTERVAL | An interval in milliseconds used to get the updated gas price value either from the oracle or from the Foreign Bridge contract.
+REACT_APP_TITLE | The title for the bridge UI page. `%c` will be replaced by the name of the network
```
Comment (andogro) [inline README.md]: add period after network.

File: README.md [hunk 3]
```diff
@@ -179,7 +179,9 @@ REACT_APP_FOREIGN_GAS_PRICE_ORACLE_URL | The URL used to get a JSON response fro
...
 REACT_APP_DESCRIPTION | The meta description for the deployed bridge page
+APP_STYLES | The set of styles to render the bridge UI page. Currently only `classic` is implemented
```
Comment (andogro) [inline README.md]: add period after implemented.

File: .env.example (hunk 1 above)
Comment (akolotov) [reply .env.example]: thanks! addressed by https://github.com/poanetwork/bridge-ui/pull/203/commits/ca343bf57cedee155e489f7fe229c6e22c8fbfb5

File: README.md (hunk 2 above)
Comment (akolotov) [reply README.md]: thanks! addressed by https://github.com/poanetwork/bridge-ui/pull/203/commits/ca343bf57cedee155e489f7fe229c6e22c8fbfb5

File: README.md (hunk 3 above)
Comment (akolotov) [reply README.md]: thanks! addressed by https://github.com/poanetwork/bridge-ui/pull/203/commits/ca343bf57cedee155e489f7fe229c6e22c8fbfb5

**Pull request merged**